src/main/java/exnihilocreatio/registries/registries/CrookRegistry.java

```diff
@@ -31,8 +29,6 @@ public CrookRegistry() {
         super(
                 new GsonBuilder()
                         .setPrettyPrinting()
-                        .registerTypeAdapter(ItemStack.class, new CustomItemStackJson())
```

Comment:
Why remove ItemStack? it is still used in the CrookReward

Reply:
Mistake, thought this was ItemInfo, not ItemStack. Readded